freeipa_auth/tests.py

```diff
@@ -180,28 +122,31 @@ def test_no_update_user(self, settings, monkeypatch,
         settings.override(FREEIPA_AUTH_ALWAYS_UPDATE_USER=False,
                           FREEIPA_AUTH_USER_FLAGS_BY_GROUP={"is_staff": ["admin"],
                                                             'is_superuser': ['superuser']})
+        # Mock user data from freeipa
+        monkeypatch.setattr("freeipa_auth.freeipa_utils.FreeIpaSession._get_user_data",
+                            lambda *args: {"givenname": ['Chester'], 'sn': ['Tester'], 'mail': ['test@enervee.com']})
 
         logged_in = self.client.login(username=self.username, password=self.password)
         assert logged_in
 
         user = User.objects.get(username=self.username)
 
-        # Assert that initially user is not a superuser
-        assert not user.is_superuser
+        # Assert that initially user does not have last name
+        assert not user.last_name
         logged_in = self.client.logout()
         assert not logged_in
 
         # Patch user groups on freeipa to have the superuser flag
         monkeypatch.setattr("freeipa_auth.freeipa_utils.FreeIpaSession.groups",
-                            ["admin", "test_group", "superuser", "test_permission"])
+                            ["admin", "test_group"])
 
         # Login again
         logged_in = self.client.login(username=self.username, password=self.password)
         assert logged_in
 
         # User should still not be superuser since FREEIPA_AUTH_ALWAYS_UPDATE_USER is set to False
```

Comment:
do these test comments match the current code?

Reply:
good point. let me modify comments. I'm not 100% that these tests actually ever run or _do_ run, which is less than ideal as well

Reply:
on second thought, a ticket has been made to address tests and make sure they actually work, which they don't right now 😅 https://enervee.atlassian.net/browse/APP-13199